lsp-multiplexer.py

```diff
@@ -506,12 +551,32 @@ def format(self, record):
         handlers=[handler]
     )
 
+    logger = logging.getLogger(__name__)
 
-    multiplexer = LSPMultiplexer([
-        ["uv", "run", "/Users/garyo/src/test-langserver/test-server.py"],
-        ["pyright-langserver", "--stdio"],
-        #"tcp://localhost:8080",
-    ])
+    # Load server configurations
+    server_configs = None
+    config_paths = [args.config] if args.config else get_default_config_paths()
+    
+    for config_path in config_paths:
+        try:
+            if config_path and os.path.exists(config_path):
+                server_configs = load_config(config_path)
+                logger.info(f"Loaded configuration from {config_path}")
+                break
+        except Exception as e:
+            logger.error(f"Error loading config from {config_path}: {e}")
+            if args.config:  # Only raise if user explicitly specified config file
+                raise
+
+    # Fall back to default configuration if no config file found
+    if not server_configs:
+        logger.info("No config file found, using default configuration")
+        server_configs = [
+            ServerConfig(command=["pyright-langserver", "--stdio"]),
+            # Add other default servers here
+        ]
+
+    multiplexer = LSPMultiplexer(server_configs)
```

Comment:
The only change I see is now you're initializing LSPMultiplexer with a list of ServerConfigs, so you should adjust the type signature of LSPMultiplexer.__init__ to reflect that. For instance:
```python
    def __init__(self, server_configs: List[ServerConfig]) -> None:
```
With that, `pyright` should pass again. (Hmm, I should set up a quick pyright test CI action.)